roles/mesos/tasks/main.yml

```diff
@@ -23,6 +23,10 @@
   include: RedHat.yml
   when: ansible_os_family == "RedHat"
 
+- name: remove old NS
+  lineinfile: dest={{mesos_dns_config_file[ansible_os_family]}} regexp="^nameserver.*" line="nameserver {{item}}" state=absent
```

Comment:
Then we better get rid of resolvconf -u and keep one template for both linux clones